src/tf_graphcycles.h

```diff
@@ -1,20 +1,25 @@
 /* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+/*******************************************************************************
 
-#ifndef NGRAPH_TF_GRAPHCYCLES_H_
-#define NGRAPH_TF_GRAPHCYCLES_H_
+This file is a copy of
+Github repository: https://github.com/tensorflow/tensorflow
+Revision: a768f270c15ded657c30fe9ef873251de3556e58
+File: tensorflow/tensorflow/compiler/jit/graphcycles/graphcycles.h
+
+*******************************************************************************/
+
+#ifndef TENSORFLOW_COMPILER_JIT_GRAPHCYCLES_GRAPHCYCLES_H_
```

Comment:
Do we want to use name "NGRAPH_TF_GRAPHCYCLES_H_" like before or stick to the name in the original file?

Reply:
It's good to change the macro as otherwise in future there may be some inclusion conflicts. Also we should "ngraph_" prefix to the filename due to the same reason.